app/views/thredded/posts_common/form/_content_field.html.erb

```diff
@@ -1,6 +1,6 @@
 <li>
   <%= form.label :content, content_label %>
-  <%= render 'thredded/posts_common/form/before_content', form: form %>
-  <%= form.text_area :content, { rows: 5, required: true }  %>
-  <%= render 'thredded/posts_common/form/after_content', form: form %>
+  <%= view_hooks.post_form.content_text_area do %>
```

Comment:
Can we have a transitional moment (maybe only during the preparation of the release version) where we have both before_content/after_content and the view hooks? Makes migration of my main_apps easier.